Neos.Neos/Classes/Command/UserCommandController.php

```diff
@@ -397,6 +425,14 @@ protected function getTableRowForUser(User $user)
                 $roleNames[] = $role->getIdentifier();
             }
         }
-        return [$user->getName()->getFullName(), $user->getPrimaryElectronicAddress(), implode(', ', $accountIdentifiers), implode(', ', $roleNames), ($user->isActive() ? 'yes' : 'no')];
+        return [
+            $user->getName()->getFullName(),
+            $user->getPrimaryElectronicAddress() ?? 'No Primary Electronic Address set',
```

Comment:
The label is a bit strange regarding the casing and stretches the table quite a bit, so I would recommend a simpler value like the other columns use:

```suggestion
            $user->getPrimaryElectronicAddress() ?? 'none',
```

Reply:
Wouldnt be: `None` better? with a capital `N`? (I'm not sure what the correct spelling would be)

Reply:
The other fallbacks are lowercase that's why I would stick with `none`.

Reply:
What about just `-`? That way we don't even need to translate it

Reply:
I would also fine with that. What do you think @Sebobo ?